19/x32/Dockerfile

```diff
@@ -3,10 +3,10 @@
 #FROM gentoo-x32
 FROM buildpack-deps-x32:jessie
 
-ENV OTP_VERSION="19.0-rc0@e038cbe"
+ENV OTP_VERSION="OTP-19.0"
 
 RUN set -xe \
-	&& OTP_DOWNLOAD_SHA1=a29ac533c61da305eb8f3e96118b40e20a524cd7 \
+	&& OTP_DOWNLOAD_SHA1=a5d3d82f8de25969094376eb40bac106b99f0dde \
```

Comment:
the x32 is a porting effort I made last October, and haven't looked into since then

it's not pushed to hub.docker.com/_/erlang/ so you don't have to take care of it, but for this time I'll take it